### PR TITLE
Document how to run an individual bootstrap test

### PR DESCRIPTION
**Pull request by eightbitraptor**
Document that the `BTESTS` environment variable can be used to run an individual bootstrap test.

### CODE REVIEW

File: doc/contributing/testing_ruby.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     make btest OPTS=-v
     ```
 
+    To run an individual bootstrap test, we can set the filename in the environment variable `BTESTS`:
+
+    ```
+    make btest BTESTS=bootstraptest/test_gc.rb
```
Comment (ko1) [inline doc/contributing/testing_ruby.md:26]: `make btest BTESTS=--sets=method` also work.

Comment (ko1) [reply doc/contributing/testing_ruby.md:26]: It's not "the environment variable".

Comment (hsbt) [reply doc/contributing/testing_ruby.md:26]: @peterzhu2118 ?

Comment (peterzhu2118) [reply doc/contributing/testing_ruby.md:26]: I opened #6194.

**Pull request merged**